docs/index.rst

```diff
@@ -19,35 +18,64 @@ Sedna documentation
 
     proposals/architecture
     proposals/dataset-and-model
+    proposals/federated-learning
+    proposals/incremental-learning
     proposals/joint-inference
+    proposals/lifelong-learning
+    proposals/object-search
+    proposals/object-tracking
 
 
 .. toctree::
-   :maxdepth: 1
-   :caption: Setup
+    :maxdepth: 1
+    :titlesonly:
+    :glob:
+    :caption: DEPLOY
 
-   setup/install
+    setup/*
```

Comment:
![image](https://user-images.githubusercontent.com/1785240/129683143-cb1ec42f-c2c3-4e47-9144-06b9b264916b.png)
There's something that wasn't displayed correctly.

Reply:
fix